app/models/post.rb

```diff
@@ -0,0 +1,23 @@
+class Post < ApplicationRecord
+  belongs_to :user, optional: true
```

Comment:
- [ ] OPTIONAL 
(_However required for your code to work as intended_)
Going by the given _ERD_, establishing the _Post_ **association** should be such that _Post_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User`.

  Kindly ensure to improve upon this aspect.